### PR TITLE
Added support for channel trailers

### PR DESCRIPTION
**Pull request by OfficialNoob**
Status: [???]

### CODE REVIEW

**Comment by ajayyy** [comment]
Also fixes same page embeds

**Comment by OfficialNoob** [comment]
@ajayyy its ready

**Comment by ajayyy** [comment]
`getYouTubeVideoID_ALT` seems to never be called

**Comment by OfficialNoob** [comment]
@ajayyy yeah I added a new function so now its used

**Comment by OfficialNoob** [comment]
Do not close I have no idea what you have changed the parser worked before

File: utils.js [hunk 1]
```diff
@@ -18,7 +18,36 @@ async function wait(condition, timeout = 5000, check = 100) {
     });
 }
 
-function getYouTubeVideoID(url) {
+// Contains both getYouTubeVideoID functions
+async function getYouTubeVideoIDAsync() {
+    let id = false;
+    id = getYouTubeVideoID();
+    if(id) return id;
+    await wait(getYouTubeVideoID_ALT).then((result) => {
```
[the remark this answers is not in the document]
Comment (OfficialNoob) [reply utils.js]: Worked when I tested it

**Comment by OfficialNoob** [comment]
@ajayyy I have tried using getYouTubeVideoID() in different places it seems to work fine Im going to try to fix it

**Comment by OfficialNoob** [comment]
Yeah why has this got mess like "SKJSADHKASDJ"

**Comment by OfficialNoob** [comment]
Chrome has built in debugging

**Comment by OfficialNoob** [comment]
@ajayyy is this inactive?

**Comment by ajayyy** [comment]
I will get back to it soon

**Comment by OfficialNoob** [comment]
Seems inactive. might be better to close this

**Pull request closed without merging**